src/pynear/account.py

```diff
@@ -8,6 +7,8 @@
 from pyonear.crypto import InMemorySigner, ED25519SecretKey
 from pyonear.transaction import Action
 
+import pynear.utils as Utils
```

Comment:
why capitalize it? 

```
from pynear import utils

utils.timestamp()
```

Reply:
It's just my style code, do you want me to update for constants also?

`import pynear.constants as Constants`

Reply:
yes